<launch>
   <group ns="bot0">
      <node pkg="argos_bridge" type="i_bug_controller.py" name="I_bug_controller" output="screen"/>
   </group>
</launch>




